docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,48 @@
+| [Exploratory Testing](guides/finding-game-breaking-bugs-with-stability-testing) | Run more than just predefined scenarios - tools such as our Monkey Bot can dynamically explore your game to find edge cases. |
+| [Composability](core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments) | Define re-usable units of bot logic that perform a range of actions such as simulating inputs, invoking MonoBehaviours, and performing assertions against the game state. Then, stitch them together with our drag-and-drop tool to form different tests.  |
+| [Runtime, CI/CD, and Editor Compatibility](core-concepts/running-tests-in-the-unity-test-runner) | Use our tools in the Editor, in builds, and within CI/CD. |
+| [Inspect and Validate Game State](core-concepts/validation-suites) | Review your recorded gameplay's screenshots, user inputs, and the internal state through our dashboard. Use these views to create test suites using no-code solutions. |
 
-This product is currently a **release preview**. We are actively talking with studios and game developers to discover the best features and use cases to tackle, and we're making changes frequently.
+## Where Do I Start?
 
-The team has focused on the foundation of supporting use cases and initial infrastructure to get bots to connect to Unity. The bulk
-of our value will come from the interfaces to build bots, the scale of our infrastructure, the speed of integration, and the products
-built around these bots.
+Before installing our SDK, you can always try our [live demo](https://play.regression.gg/validations-demo). Once you are ready to hop in, follow the
+[installation guide](getting-started/installing-regression-games), after which you can 
+[create your first automated test](getting-started/creating-your-first-automated-test).
 
-![timeline feature](unused-content/session-insights/img/replay/timeline.png)
+If you are already familiar with our framework, check out these more advanced guides:
 
-## Availability Assurances
+* [Interacting with User Interfaces using Computer Vision](guides/interacting-with-ui-using-computer-vision)
+* [Finding Game-Breaking Bugs with Stability Testing](guides/finding-game-breaking-bugs-with-stability-testing)
+* [Writing Advanced Validations](guides/writing-advanced-validations)
+* [Running Tests in Editor and in Builds](guides/running-tests-in-editor-and-in-builds)
 
-The Regression Games team offers the following assurances for our initial testing with developers. Additionally, we have a dedicated Discord server for support, and can also create a Slack Connect channel, giving you direct access to our entire team. For access to
-these channels, send an e-mail to [aaron@regression.gg](mailto:aaron@regression.gg).
+You can also check out our [existing game samples](additional-game-samples).
```

Comment:
what is the additional-game-samples page eventually going to contain / how will it help users? It'd be a good idea to throw that into this sentence ("You can check out our existing game samples for x")

Reply:
Made this a little more clear. Can update further as well once we add more samples!